controllers/grid/users/stageParticipant/StageParticipantGridRow.inc.php

```diff
@@ -84,14 +86,37 @@ function initialize($request, $template = null) {
 			) {
 				$dispatcher = $router->getDispatcher();
 				import('lib.pkp.classes.linkAction.request.RedirectConfirmationModal');
+				$userGroupDAO = DAORegistry::getDAO('UserGroupDAO');
+				$userGroup = $userGroupDAO->getById($userGroupId, $context->getId()); 
+
+				if ($userGroup->getRoleId() == ROLE_ID_AUTHOR) {
```

Comment:
@NateWr, I suspect you could probably recommend a helper to take the place of this if/else...

Reply:
Hmm, nothing's coming to me off the top of my head. I think this looks alright.